src/components/SearchBar.tsx

```diff
@@ -246,7 +255,12 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
                 <div className="SearchInput">
                     <div>
                         <i className="fa fa-search SearchBarIcon"></i>
-                        <input className="SearchInputText" type="input" placeholder="Search..." onChange={this.onTextChanged.bind(this)} ref={(field) => { this.searchInputField = field; this.props.setSearchInputField(field) }}></input>
+                        <input
+                            className="SearchInputText"
+                            type="input" placeholder="Search..."
+                            onChange={this.onTextChanged.bind(this)}
+                            ref={(field) => { this.searchInputField = field; this.props.setSearchInputField(field) }}>
+                        </input>
```

Comment:
Hi @fanwgwg can you make a new task to remove `setSearchInputField` from `props`? Unless I'm wrong but I can't imagine why would external components need to have access to the text box (as opposed to dealing with it through a set of APIs exposed from `SearchBar` component). Please add the task and put the link here (and make that task reference to this particular PR comment).